### PR TITLE
Included link to https://www.tammacarleton.com/

### PR DESCRIPTION
**Pull request by tliddell4**
All that's changed is one blurb in the readme that links to Tamma's website

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -20,6 +20,8 @@ knitr::opts_chunk$set(
 
 The R Package `stagg` aims to harmonize the preparation of spatiotemporal data obtained from the [ERA5 project](https://www.ecmwf.int/en/forecasts/datasets/reanalysis-datasets/era5) for use in statistical analyses. This allows for more efficient useof researchers' time, and avoids common missteps. The end goal is a greater quantity of quality research into the study of coupled human-geophysical systems. 
 
+Further documentation, including the working paper, AGU conference poster, and cheat sheet are available [here](https://www.tammacarleton.com/)
```
Comment (tcarleton) [inline README.Rmd:23]: @tliddell4 please change URL to https://www.tammacarleton.com/projects-6

**Review by tcarleton — CHANGES_REQUESTED** [review]

File: README.Rmd (hunk 1 above)
Comment (traceybit) [reply README.Rmd]: @tcarleton i changed the URL and added to the gitignore.

File: .gitignore
Comment (traceybit) [inline .gitignore:1]: it looks like this is already on line 1 -- any reason for adding it again?

Comment (traceybit) [inline .gitignore:1]: looks like **/.DS_Store is already on line 1 -- any reason for adding it to line 6?

**Review by traceybit — CHANGES_REQUESTED** [review]
i did my best to review although there are a lot of changes!! the only thing that i noticed that i think we should check is that i think now there are two DS_stores in the gitignore.

File: .gitignore
Comment (saraorofino) [reply .gitignore:1]: Done - just deleted the duplicate in the gitignore

**Review by traceybit — APPROVED** [review]
i reviewed the text and previously reviewed the code -- approving!

**Review by traceybit — COMMENTED** [review]
website updated

**Pull request merged**